cli/command/cli.go

```diff
@@ -271,6 +276,27 @@ func newHTTPClient(host string, tlsOptions *tlsconfig.Options) (*http.Client, er
 	}, nil
 }
 
+func tlsConfigFromOptions(opts *cliflags.CommonOptions) (*tlsconfig.Options, error) {
+	if opts.TLSOptions == nil {
+		return nil, nil
+	}
+
+	tlsMinVersion, err := dopts.ParseTLSMinVersion(opts.TLSOptions.MinVersion)
+	if err != nil {
+		return nil, err
+	}
+
+	tlsOptions := &tlsconfig.Options{
+		CAFile:             opts.TLSOptions.CAFile,
+		CertFile:           opts.TLSOptions.CertFile,
+		KeyFile:            opts.TLSOptions.KeyFile,
+		InsecureSkipVerify: !opts.TLSVerify,
```

Comment:
Is `opts.TLSVerify` a new option? It would be great to invert the expected value (and necessarily change the name) to take advantage of Go's default zero values. Defaults should be secure but in this case, the default zero value, `false`, would cause `InsecureSkipVerify` to be `true`.